packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -42,10 +42,11 @@ interface ListViewItemProps< Item > {
 	idPrefix: string;
 	isSelected: boolean;
 	item: Item;
+	titleField?: NormalizedField< Item >;
 	mediaField?: NormalizedField< Item >;
+	descriptionField?: NormalizedField< Item >;
 	onSelect: ( item: Item ) => void;
-	primaryField?: NormalizedField< Item >;
-	visibleFields: NormalizedField< Item >[];
+	otherFields: NormalizedField< Item >[];
```

Comment:
Should otherFields be regularFields to be consistent with the grid config?

Reply:
In grid, there's two types "badge and regular" that's why I split it there.